src/client/src/components/TasksList/__tests__/tasksList.test.jsx

```diff
@@ -0,0 +1,15 @@
+import renderer from "react-test-renderer";
+import TasksList from "./../TaskListConnector";
+import { store } from "./../../../redux/store";
+import { Provider } from "react-redux";
+
+it("renders correctly", () => {
```

Comment:
according to the instructions you should have created a snapshot test for taskItem and not taskList
![image](https://user-images.githubusercontent.com/22417492/179469153-f1aa0f5e-b0fc-4039-a609-c574f8b0a412.png)